src/miniasync.def

```diff
@@ -13,3 +13,4 @@ EXPORTS
     vdm_delete
     vdm_descriptor_threads
     vdm_descriptor_threads_polled
+    os_rand_r
```

Comment:
This change cannot take place

Reply:
I've removed it and added files from the core as a static library, which is then included in the miniasync library and in tests.